scss/_variables.scss

```diff
@@ -44,7 +44,7 @@ $popup-padding-x: 0 !default;
 $popup-padding-y: 0 !default;
 
 $header-padding-x: $padding-x !default;
-$header-padding-y: $padding-y !default;
+$header-padding-y: $padding-x !default;
```

Comment:
In terms of values, `x` and `y` must be equal for theme-default. In terms of variables, let's keep both `x` and `y` separate to allow easy and predictable customization.